docs/modules/ROOT/pages/overview.adoc

```diff
@@ -0,0 +1,32 @@
+= Hazelcast JDBC Driver
+:url-github-jdbc: https://github.com/hazelcast/hazelcast-jdbc
+:description: Java Database Connectivity (JDBC) is the Java API standard for managing connections to databases, executing queries on them, and retrieving results. Hazelcast's JDBC driver allows you to connect to and execute SQL queries on Hazelcast clusters using application frameworks, libraries, and tools that support JDBC. 
+
+{description}
+
+You can use the Hazelcast JDBC driver with both local Hazelcast clusters and remote clusters deployed on public cloud platforms.
+
+NOTE: Currently, the Hazelcast JDBC driver does not support connections to Hazelcast {hazelcast-cloud}. Support is planned for upcoming releases.
```

Comment:
I think we do: https://github.com/hazelcast/hazelcast-jdbc/blob/69a1f8deaea6e498fe262bb49c2363de9f6ae7ad/README.md?plain=1#L146-L148

Reply:
@viliam-durina. I tried this but it doesn't seem to work.